plugins/infinispan100/src/main/java/org/radargun/service/ServerLogPattern.java

```diff
@@ -9,7 +9,7 @@
 public enum ServerLogPattern {
 
    START_OK(".*\\[org\\.infinispan\\.SERVER\\].*started in.*"),
-   START_ERROR(".*\\[org\\.infinispan\\.SERVER\\].*Infinispan Server stopping.*"),
+   START_ERROR(".*\\[org\\.infinispan\\.SERVER\\].*Server failed to start.*"),
```

Comment:
@diegolovison How did you get this message?
If you start the server and try to start another instance without change the port, the server will not start, but you will not see `Server failed to start`

Reply:
You need https://github.com/infinispan/infinispan/pull/7449

Reply:
@diegolovison lol it's not fair, this PR was merged minutes ago! I'm testing again

Reply:
After pull and compile upstream with the last modifications I still receive the same messages.
- `pull ispn master`
- `mvn clean install ... `
- run `server/runtime/target/infinispan-server-10.0.0-SNAPSHOT/bin/server.sh`
- run `server/runtime/target/infinispan-server-10.0.0-SNAPSHOT/bin/server.sh` again